File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if len(s) != len(t):
            return False
        freq = [0]*26

        for i in range(len(s)):
            freq[ord(s[i])-ord('a')] +=1
            freq[ord(t[i])-ord('a')] -=1
        

        for value in freq:
            if value != 0:
                return False
        
        return True
        

File: 0242-valid-anagram/0242-valid-anagram.go
func isAnagram(s string, t string) bool {
    if len(s) != len(t){
        return false
    }

    freq := [26]int{}

    for i:=0;i<len(s);i++{
        freq[s[i]-'a']++
        freq[t[i]-'a']--
    }

    for _, val := range freq {
        if val != 0 {
            return false
        }
    }

    return true
}